app/javascript/dashboard/routes/dashboard/contacts/components/Header.vue

```diff
@@ -142,4 +155,7 @@ export default {
     visibility: visible;
   }
 }
+.margin-right-small {
```

Comment:
Can we create helper functions globally instead of having them here in the component?

Reply:
Moved utility helper classes to a new file in https://github.com/chatwoot/chatwoot/commit/5c8304e303217f7cec9869fcc0649401a1d9c1b9